requirements.txt

```diff
@@ -58,7 +58,7 @@ markdown==3.2.2           # via apache-superset (setup.py)
 markupsafe==1.1.1         # via jinja2, mako, wtforms
 marshmallow-enum==1.5.1   # via flask-appbuilder
 marshmallow-sqlalchemy==0.23.0  # via flask-appbuilder
-marshmallow==2.21.0       # via flask-appbuilder, marshmallow-enum, marshmallow-sqlalchemy
+marshmallow==3.6.0        # via flask-appbuilder, marshmallow-enum, marshmallow-sqlalchemy
```

Comment:
This caught my eye, `3.6.1` is now out: https://pypi.org/project/marshmallow/3.6.1/

Reply:
Good point, checked the changelog and it's just docs, safe bump ;)
https://github.com/marshmallow-code/marshmallow/blob/dev/CHANGELOG.rst#361-2020-06-02